src/Microsoft.Azure.Relay/Common/InputQueue.cs

```diff
@@ -470,11 +463,9 @@ bool EnqueueWithoutDispatch(Item item)
                         itemQueue.EnqueueAvailableItem(item);
                         return false;
                     }
-                    else
-                    {
-                        itemQueue.EnqueuePendingItem(item);
-                        return true;
-                    }
+
+                    itemQueue.EnqueuePendingItem(item);
+                    return true;
```

Comment:
This is really just a minor style preference one way or another. Though less indentation is generally good IMO so it's fine.

Reply:
yes I think is good if  {}  if {} or if {}  return. In others repos I've contributed the team tried to avoid `else` specifically